File: LootMarket Backend/src/main/java/com/example/lootmarketbackend/dto/UtenteDTO.java
package com.example.lootmarketbackend.dto;

import com.example.lootmarketbackend.Models.Indirizzo;

public class UtenteDTO {
    public String nome;
    public String codiceFiscale;
    public String mail;
    public int dataDiNascitaAnno;
    public int dataDiNascitaMese;
    public int dataDiNascitaGiorno;
    public String nazione;
    public String numeroCellulare;
    public Indirizzo indirizzoSpedizione;
    public Indirizzo indirizzoFatturazione;
    public String sito;
    public String socialFacebook;
    public String socialInstagram;
    public String biografia;
    public String ragioneSociale;
    public String partitaIva;
    public String numeroAziendale;
    public String immagineProfilo;

    public UtenteDTO(String nome, String codiceFiscale, String mail, int dataDiNascitaAnno, int dataDiNascitaMese, int dataDiNascitaGiorno, String nazione, String numeroCellulare, Indirizzo indirizzoSpedizione, Indirizzo indirizzoFatturazione, String sito, String socialFacebook, String socialInstagram, String biografia, String ragioneSociale, String partitaIva, String numeroAziendale, String immagineProfilo) {
        this.nome = nome;
        this.codiceFiscale = codiceFiscale;
        this.mail = mail;
        this.dataDiNascitaAnno = dataDiNascitaAnno;
        this.dataDiNascitaMese = dataDiNascitaMese;
        this.dataDiNascitaGiorno = dataDiNascitaGiorno;
        this.nazione = nazione;
        this.numeroCellulare = numeroCellulare;
        this.indirizzoSpedizione = indirizzoSpedizione;
        this.indirizzoFatturazione = indirizzoFatturazione;
        this.sito = sito;
        this.socialFacebook = socialFacebook;
        this.socialInstagram = socialInstagram;
        this.biografia = biografia;
        this.ragioneSociale = ragioneSociale;
        this.partitaIva = partitaIva;
        this.numeroAziendale = numeroAziendale;
        this.immagineProfilo = immagineProfilo;
    }
}

File: LootMarket/app/src/main/java/com/danilo/lootmarket/Network/dto/UtenteDTO.kt
package com.danilo.lootmarket.network.dto

import com.danilo.lootmarket.modelview.IndirizzoView

data class UtenteDTO(
    var nome: String,
    var codiceFiscale: String,
    var mail: String,
    var dataDiNascitaAnno: Int,
    var dataDiNascitaMese: Int,
    var dataDiNascitaGiorno: Int,
    var nazione: String,
    var numeroCellulare: String,
    var indirizzoSpedizione: IndirizzoView,
    var indirizzoFatturazione: IndirizzoView,
    var sito: String,
    var socialFacebook: String,
    var socialInstagram: String,
    var biografia: String,
    var ragioneSociale: String,
    var partitaIva: String,
    var numeroAziendale: String,
    var immagineProfilo: String
)
